src/Elmish/Browser.purs

```diff
@@ -0,0 +1,37 @@
+module Elmish.Browser
+    ( sandbox
+    , module Exported
+    ) where
+
+import Prelude
+
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Effect.Exception (throw)
+import Elmish.Component (ComponentDef, construct)
+import Elmish.Dispatch (dispatchMsgFn)
+import Elmish.React.DOM as ReactDOM
+import Web.DOM.ParentNode (QuerySelector(..)) as Exported
+import Web.DOM.ParentNode (QuerySelector(..), querySelector)
+import Web.HTML (window) as HTML
+import Web.HTML.HTMLDocument (toParentNode) as HTMLDocument
+import Web.HTML.Window (document) as HTML
+
+sandbox :: forall msg state. QuerySelector -> ComponentDef Aff msg state -> Effect Unit
+sandbox selector@(QuerySelector rawSelector) component = do
```

Comment:
Also, I don't think this is a good name: it creates the feeling of "testing" or "limiting capabilities" - as if you would want to put your component in a sandbox for a while before it's ready for production use.

Reply:
See: https://github.com/collegevine/purescript-elmish/pull/6#discussion_r285642172 + https://package.elm-lang.org/packages/elm/browser/latest/Browser